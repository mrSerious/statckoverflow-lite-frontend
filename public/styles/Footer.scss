/*
* footer
 */
.footer {
  background: $footer-bg;
  bottom: 0;
  color: $white;
  flex-shrink: 0;
  height: 10rem;
  line-height: 10rem;
  text-align: center;
}
