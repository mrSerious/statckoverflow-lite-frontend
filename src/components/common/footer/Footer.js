import React from 'react';

const Footer = () => (
  // Footer
  <footer className="footer">
    <p className="footer-company-name no-margin">StackOver-Lite &copy; 2018</p>
  </footer>
);

export default Footer;
