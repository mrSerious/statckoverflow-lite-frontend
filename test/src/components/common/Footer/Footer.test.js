import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../../../../src/components/common/footer/Footer';

test('App snapshot test', () => {
  const component = shallow(<Footer />);
  expect(component).toMatchSnapshot();
});
