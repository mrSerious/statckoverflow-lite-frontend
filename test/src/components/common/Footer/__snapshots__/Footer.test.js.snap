// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<footer
  className="footer"
>
  <p
    className="footer-company-name no-margin"
  >
    StackOver-Lite Â© 2018
  </p>
</footer>
`;
